### ISSUE TITLE
Add the ability to follow an author

### ISSUE THREAD

**Issue by nedtwigg** [mytakedotorg/mytakedotorg#163]
- [x] Create a migration script for the database in `server/src/main/resources/db/migration`

My guess is you want to create a table along these lines:

```sql
CREATE TABLE follow (
    author int REFERENCES account (id),
    follower int REFERENCES account (id),
    PRIMARY KEY (author, follower), --can only follow a given author once
    followed_at timestamp NOT NULL
)
```

- [x] run `gradlew jooq` to generate new code under `jooq-generated`
- [x] in the `controllers.Profile` class, implement the `case followers/following`
https://github.com/mytakedotorg/mytakedotorg/blob/f1020d213e43bf6817abab37e519bced6b52b3bc/server/src/main/java/controllers/Profile.java#L109-L110
- [x] add a "follow" button for following a user
- [x] create some tests

**Comment by nedtwigg** [comment]
See #168 for a good way to test this kind of thing.  Also, if you see a way to fix #168 while you're working on this, that'd be great :)

**Comment by WebsByTodd** [comment]
The console logs a stack trace when running `gradlew jooq`. Does this happen for you too? The command works and the output is generated, there's just a weird error logged.

```
$: ./gradlew jooq
Parallel execution is an incubating feature.
while reading output
java.io.IOException: Stream closed
        at java.io.BufferedReader.ensureOpen(BufferedReader.java:122)
        at java.io.BufferedReader.readLine(BufferedReader.java:317)
        at java.io.BufferedReader.readLine(BufferedReader.java:389)
        at com.opentable.db.postgres.embedded.ProcessOutputLogger.run(ProcessOutputLogger.java:48)
        at java.lang.Thread.run(Thread.java:745)

BUILD SUCCESSFUL in 3s
1 actionable task: 1 executed
```

**Comment by nedtwigg** [comment]
I get this error also, it's not a problem.  It's just the thread that dumps the postgres stdout to gradle's stdout.  Hopefully one of these days when we bump our upstream dependencies it'll magically get fixed :)

**Comment by WebsByTodd** [comment]
Implemented and fixed #168. Merged into master.
